src/Symfony/Component/Serializer/Mapping/Loader/AnnotationLoader.php

```diff
@@ -176,7 +175,7 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata): bool
         return $loaded;
     }
 
-    public function loadAnnotations(\ReflectionMethod|\ReflectionClass|\ReflectionProperty $reflector): iterable
```

Comment:
I wonder why this method is public anyway. All calls come from within this class. Let's just deprecate it without replacement. The new `loadAttributes()` method can be private.